documentation/book/con-reassignment-of-partitions.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// assembly-scaling-clusters.adoc
+
+[id='con-reassignment-of-partitions-{context}']
+
+= Reassignment of partitions
+
+The Topic Operator does not currently support reassigning replicas to different brokers, so it is necessary to connect directly to broker pods to reassign replicas to brokers.
+
+Within a broker pod the `kafka-reassign-partitions.sh` utility is used to reassign partitions to different brokers.
+
+It has three different modes:
+
+`--generate`::
+Takes a set of topics and brokers and generates a _reassignment JSON file_ which will result in the partitions of those topics being assigned to those brokers.
+It is an easy way to generate a _reassignment JSON Because {ProductName} uses `StatefulSets` to manage the broker pods you cannot remove _any_ pod from the cluster. You can only remove the highest numbered pod(s) from the cluster.file_, but it operates on whole topics, so its use is not always appropriate.
+
+`--execute`::
+Takes a _reassignment JSON file_ and applies it to the partitions and brokers in the cluster.
+Brokers which are gaining partitions will become followers of the partition leader.
+For a given partition, once the new broker has caught up and joined the ISR the old broker will stop being a follower and will delete its replica.
+
+`--verify`::
+Using the same _reassignment JSON file_ as the `--execute` step, `--verify` checks whether all of the partitions in the file have been moved to their intended brokers.
+If the reassignment is complete it will also remove any xref:con-reassignment-throttles-{context}[throttles] which are in effect.
```

Comment:
If the reassignment is complete, `--verify` also removes...

which are in effect > that are in effect